### DISCUSSION TITLE
problem enable mask upload

### DISCUSSION THREAD

**Discussion by jackalxp5b** [lllyasviel/Fooocus#3474] [Q&A] [answered]
Hi to the whole community, before the new software update, I used a feature that allowed me to obtain from 2 images, one in color and the other darkened, through the description in the prompt, a figure painted on the target clothes that I was interested in didn't change anything, and I used the enable mask upload function, now I can't find it in the tab anymore... but how can I get the same result as before for clothes?

**Comment by mashb1t** [comment, marked as answer]
the option for mask upload has been moved directly to the inpaint tab and is now called "enable advanded mask features".

**Comment by Yogesh-DevHub** [comment]
> Hi to the whole community, before the new software update, I used a feature that allowed me to obtain from 2 images, one in color and the other darkened, through the description in the prompt, a figure painted on the target clothes that I was interested in didn't change anything, and I used the enable mask upload function, now I can't find it in the tab anymore... but how can I get the same result as before for clothes?

Hi, the Fooocus 2.5.0 had some significant changes which included mask feature. 
You can refer the details here: https://github.com/lllyasviel/Fooocus/discussions/3281 

There is this short comprehensive video which cover the major new features of Fooocus 2.5.0 including mask https://youtu.be/4oWGtIHFlCQ?t=517&si=BQA93-42JbwHJZVF
